rack-mount/catalog/beelink-EQi12-tray_mod.scad

```diff
@@ -0,0 +1,151 @@
+use <../tray/tray.scad>
+
+/*
+  Simple tray for a beelink mini-s series mini pc.
+
+  Please make sure to configure the correct rack frame dimensions in rackFrame.scad.
+*/
+
+difference () {
+    bottomScrewTray (
+        u = 5,
+        trayWidth = 140,
+        trayDepth = 140,
+        trayThickness = 3,
+        frontLipHeight = 3,
+        backLipHeight = 5,
+        /*mountPoints = [[20, 70], [20+95.25, 70]],*/

```

Comment:
is this intentional? we should probably have mount points right?

Reply:
EQi12 does not have mounting points. I have removed this line from the code completely in the next upload.